recipes/libnice/all/conanfile.py

```diff
@@ -5,6 +5,7 @@
 from conan.tools.layout import basic_layout
 from conan.tools.gnu import PkgConfigDeps
 from conan.tools.microsoft import is_msvc_static_runtime
+from conan.tools.apple import fix_apple_shared_install_name
 from conan.errors import ConanInvalidConfiguration
 
```

Comment:
```suggestion

required_conan_version = ">=1.60.0"
```

https://docs.conan.io/1/changelog.html?highlight=host_version#id4